lms/lmsweb/views.py

```diff
@@ -29,6 +28,8 @@
 from lms.utils.consts import RTL_LANGUAGES
 from lms.utils.files import get_language_name_by_extension
 from lms.utils.log import log
+from lms.lmsweb.manifest import MANIFEST
```

Comment:
This import line should appear after `from lms.lmsweb.config import LANGUAGES, LOCALE`